spark-indexer/src/test/java/org/uniprot/store/spark/indexer/common/util/SolrUtilsTest.java

```diff
@@ -18,7 +18,7 @@ void convertToSolrInputDocumentSuccess() {
 
         SolrInputDocument solrInput = SolrUtils.convertToSolrInputDocument(doc);
         assertNotNull(solrInput);
-        assertEquals(6, solrInput.values().size());
+        assertEquals(7, solrInput.values().size());
```

Comment:
why 1 more now?

Reply:
Because i had to replace content field with two new fields synonyms and definition. Earlier we used to keep these values in content field.